sunpy/net/jsoc/attrs.py

```diff
@@ -1,11 +1,16 @@
 from __future__ import absolute_import
 
-from sunpy.net.attr import AttrWalker, AttrAnd, AttrOr
-from sunpy.net.vso.attrs import _VSOSimpleAttr
-from sunpy.net.vso.attrs import Time, Wavelength
+from sunpy.net.attr import AttrWalker, AttrAnd, AttrOr, Attr
+from sunpy.net.vso.attrs import _VSOSimpleAttr, _Range
+from sunpy.net.vso.attrs import Time as VSOTime
+from sunpy.net.vso.attrs import Wavelength
 
+from sunpy.time import parse_time
+from sunpy.time import TimeRange as _TimeRange
 
-__all__ = ['Series', 'Protocol', 'Notify', 'Compression', 'Segment']
+from astropy.time import Time as astropyTime
```

Comment:
the order of imports is not correct, astropy should go between `__future__` and  sunpy's